general/payment_acquirers.rst

```diff
@@ -7,5 +7,5 @@ Payment Acquirers
 
    payment_acquirers/payment_acquirers
    payment_acquirers/wire_transfer
-   payment_acquirers/paypal
    payment_acquirers/authorize
+   payment_acquirers/paypal
```

Comment:
any reason for this switch?

Reply:
alpahbetical, after the bank wire.
The logic would be 
- bank payment acquirers
- online payment providers (alphabetical)